package.json

```diff
@@ -8,20 +8,22 @@
   "workspaces": {
     "packages": [
       "packages/*"
-    ],
-    "nohoist": [
-      "@graduate/frontend/**"
     ]
   },
   "scripts": {
-    "dev": "concurrently -n \"common,api,frontend\" \"yarn workspace @graduate/common dev\" \"yarn workspace @graduate/api dev\" \"yarn workspace @graduate/frontend dev\"",
-    "dev:v2": "concurrently -n \"common,api-v2,frontend,proxy\" \"yarn workspace @graduate/common dev\" \"yarn workspace @graduate/api-v2 dev\" \"yarn workspace @graduate/frontend dev\" \"yarn dev:proxy\"",
+    "dev": "concurrently -n \"common,api,frontend\" \"yarn packages/common dev\" \"yarn packages/api dev\" \"yarn packages/frontend dev\"",
+    "dev:v2": "concurrently -n \"common,api-v2,frontend,proxy\" \"yarn packages/common dev\" \"yarn packages/api-v2 dev\" \"yarn packages/frontend dev\" \"yarn dev:proxy\"",
     "dev:proxy": "node infrastructure/develop/dev-proxy.js",
     "dev:db:up": "docker-compose -f infrastructure/develop/docker-compose.yml up -d",
     "dev:db:down": "docker-compose -f infrastructure/develop/docker-compose.yml down",
-    "prod:start": "concurrently \"yarn workspace @graduate/api dev\" \"yarn workspace @graduate/frontend docker:start\"",
+    "prod:start": "concurrently \"yarn packages/api dev\" \"yarn packages/frontend docker:start\"",
```

Comment:
oh just curious though i haven't run it - why do we not need to run `yarn packages/common` here? Is it cuz that package is just for dev usage?

Reply:
good question! So currently common package is only being used by the frontend(the old backend doesn't use it). The `prod:start` command run the frontend in docker. The Dockerfile for the frontend is responsible for copying over the common directory along with the frontend and building them together!

Eventually this command will run the new backend with the frontend... if we decide to Dockerize our new backend then the Dockerfile for the backend will copy over the backend + common + any other dependency package and build them all. But this is far out haha.

Reply:
ah gotcha thankss